tests/test_bookmarks.py

```diff
@@ -48,8 +48,13 @@ def bookmarks_test(self, conn_id, testable_streams):
 
         # Select all streams and no fields within streams
         found_catalogs = menagerie.get_catalogs(conn_id)
+        # `transactions` is child stream of `orders` stream which is incremental.
+        # We are writing a separate bookmark for the child stream in which we are storing 
+        # the bookmark based on the parent's replication key.
+        # But, we are not using any fields from the child record for it.
+        # That's why the `transactions` stream does not have replication_key but still it is incremental.
         incremental_streams = {key for key, value in self.expected_replication_method().items()
-                               if value == self.INCREMENTAL and key in testable_streams}
+                               if value == self.INCREMENTAL and key in testable_streams and key not in ('transactions')}
 
         # Our test data sets for Shopify do not have any abandoned_checkouts
```

Comment:
The abandoned_checkouts stream should remain under test. It is incorrectly removed on crest master. See base.py init

Reply:
The abandoned checkouts are added back in another PR in this [commit](https://github.com/singer-io/tap-shopify/pull/134/commits/7bfc94ec45bbc35ba61d3ea7a299a55263883474)